shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1729,6 +1735,7 @@ - (void)scrollEvent:(UIPanGestureRecognizer*)recognizer API_AVAILABLE(ios(13.4))
 - (void)encodeRestorableStateWithCoder:(NSCoder*)coder {
   NSData* restorationData = [[_engine.get() restorationPlugin] restorationData];
   [coder encodeDataObject:restorationData];
+  [super encodeRestorableStateWithCoder:coder];
```

Comment:
This was an analyzer warning, `encodeRestorableStateWithCoder` needs to encode the super class's state.